test/cli/windows-line-endings/windows-line-endings.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+file --brief "$(realpath test/cli/windows-line-endings/windows-line-endings.rb)"
```

Comment:
@neilparikh the realpath is redundant here, right? Can we drop it if so?

Reply:
`test/cli/windows-line-endings/windows-line-endings.rb` is a symlink, so without `realpath`, `file` prints something like `symbolic link to...` on Linux, rather than the line endings.